static/css/components/search-result-item.less

```
@@ -117,7 +117,7 @@
     }
   }
   a:hover span {
-    background-position: 21px 0;
+    background-position: right 0;
```

```suggestion
    background-position: -21px 0;
```
Let's be explicit with the position.  It looks like this was originally `-21px`, but the `-` was lost during a refactor: https://github.com/internetarchive/openlibrary/pull/1838/files#diff-f8e75fa1ff234c063725cfc97460f89e6d1be3fe9d192140663e035efbad3075L2982

---

Thank you for the review.
Also the idea of looking back in commit history to find the same code is a good idea. I will try that with my other issues as well.
Have a good day.